src/components/Widgets/DateControl.js

```diff
@@ -1,28 +1,56 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import DateTime from 'react-datetime';
+import moment from 'moment';
 
-export default class DateControl extends React.Component {
+function format(format, value) {
```

Comment:
I'm not a fan of the interface of `format` and `toDate` - depending on the falsiness of one argument they:

- either do something or do nothing
- either return either a specific type of value or return whatever was passed in